contracts/ThalesRoyale/ThalesRoyale.sol

```diff
@@ -106,8 +106,9 @@ contract ThalesRoyale is Initializable, ProxyOwned, PausableUpgradeable, ProxyRe
 
     /* ========== GAME ========== */
 
-    function signUp() external {
+    function signUp(uint position) external {
```

Comment:
position should be optional

Reply:
you mean:

0 - player did't set value
1 and 2 (DOWN and UP)

but parameter stay

Reply:
done.